src/components/TaskclusterAuth/loader.ts

```diff
@@ -20,11 +28,22 @@ export function loader({ request }: { request: Request }) {
     );
   }
 
-  const tokenBearer = retrieveTaskclusterToken(rootUrl, taskclusterCode);
+  const tokenResponse = await retrieveTaskclusterToken(
+    rootUrl,
+    taskclusterCode,
+  );
+
+  storeToken({ [tokenResponse.token_type]: tokenResponse });
```

Comment:
This should look like this:
```suggestion
  storeToken({ [rootUrl]: tokenResponse });
```

but please look at my other comments as well, as I believe the signature will be changed and therefore this will look like this:
```suggestion
  storeToken(rootUrl, tokenResponse);
```